addons/l10n_tw_edi_ecpay_pos/data/res_partner_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="ecpay_default_walk_in_customer" model="res.partner">
+            <field name="name">Walk In Customer</field>
+            <field name="email">walkin@test.com</field>
```

Comment:
Don't add a fake/test email in master data.

Also, test.com is definitely not safe to use /!\ it's privately owned.
Use test.test or example.com if needed (but only in demo data, not in master data)